### DISCUSSION TITLE
What is the best way to make a button full width on just the mobile break point

### DISCUSSION THREAD

**Discussion by Catchpowle** [chakra-ui/chakra-ui#3349] [Q&A] [answered]
The isFullWidth prop isn't responsive so would using `width={{ base: '100%', sm: 'auto' }}` be the best way to go about this? 🤔

**Comment by dodas** [comment, marked as answer]
Hey @Catchpowle,

yes, using `width` is absolutely fine for this, as that's exactly what `isFullWidth` prop does under the hood.

We should either add support for responsive values for `isFullWidth`, or deprecate it in favor of just using `width`, similarly to [`Input`](https://github.com/chakra-ui/chakra-ui/blob/6456f531a1c6bcc4873bd69033b80a45405f8b25/packages/input/src/input.tsx#L29).

**Reply by dedane** [reply]
Did you get this functionality working
